customer_account_test.go

```diff
@@ -344,7 +344,10 @@ func TestCustomerAccountListContactsPaginateAndFilter(t *testing.T) {
 
 	customerAccountApi := CustomerAccountApi{}
 	ctx := context.Background()
-	resp, err := customerAccountApi.ListContacts(ctx, 1, 5, []string{"GENERAL", "SECURITY", "TECHNICAL", "BILLING"})
+	opts := CustomerAccountContactsListOptions{
+		PrimaryRoles: String("GENERAL, SECURITY, TECHNICAL, BILLING"),
```

Comment:
Same here, shouldn't this be a `[]string`?

Reply:
Done.